Sources/PaymentsCore/Networking/Enums/HTTPMethod.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public enum HTTPMethod: String {
```

Comment:
We likely won't need all of these. Maybe let's include the ones we need for now, and then add the others as the need arises.